project/src/main/resources/templates/cart.html

```diff
@@ -48,7 +48,9 @@ <h1>Your Active Cart</h1>
           <div class="text-right text-primary h4">Total: $<span id="total"></span></div>
           
           <div class="text-left">
-            <button id="checkout" type="button" class="btn btn-primary">Proceed to Checkout</button>
+            <a href="/checkout">
+              <button id="checkout" type="button" class="btn btn-primary">Proceed to Checkout</button>
```

Comment:
@BoxThatBeat What would the best way to send the POST request from this button?
I tried using a html form like the logout button uses but didnt seem to work and also created a separate JS file and added an event listener but didn't work that well.

I'm having trouble properly sending the HttpSession object to the post request to have it do its thing.

Reply:
Managed to get it working using a form and simple post action.